CRM/Mailing/Page/View.php

```diff
@@ -113,7 +113,14 @@ public function run($id = NULL, $contactID = NULL, $print = TRUE, $allowID = FAL
       $this->_mailing = new CRM_Mailing_BAO_Mailing();
 
       if (!is_numeric($this->_mailingID)) {
+
+        //lets get the id from the hash
+        $result_id = civicrm_api3('Mailing', 'get', [
+          'return' => ["id"],
```

Comment:
minor stylistic thing - using single quotes whenever double quotes are not needed is preferred

Reply:
noted. I have updated the code.